dist/vue-tabs.common.js

```diff
@@ -195,6 +195,12 @@ var VueTabs = {
                     [_this.textPosition === 'top' && _this.renderTabTitle(index, _this.textPosition), h(
                         'a',
                         {
+                            on: {
```

Comment:
This is a built file. Please do not commit it but rather modify the files inside `src` folder and commit them instead.

Reply:
yep, my fault, i will change this.